nixos/tests/musescore.nix

```diff
@@ -40,51 +39,79 @@ in
     # Inject custom settings
     machine.succeed("mkdir -p /root/.config/MuseScore/")
     machine.succeed(
-        "cp ${customMuseScoreConfig} /root/.config/MuseScore/MuseScore3.ini"
+        "cp ${customMuseScoreConfig} /root/.config/MuseScore/MuseScore4.ini"
     )
 
     # Start MuseScore window
     machine.execute("DISPLAY=:0.0 mscore >&2 &")
 
     # Wait until MuseScore has launched
-    machine.wait_for_window("MuseScore")
+    machine.wait_for_window("MuseScore 4")
 
     # Wait until the window has completely initialised
-    machine.wait_for_text("MuseScore")
+    machine.wait_for_text("MuseScore 4")
+
+    machine.screenshot("MuseScore0")
+
+    # Create a new score
+    machine.send_key("ctrl-n")
+
+    # Wait until the creation wizard appears
+    machine.wait_for_window("New score")
+
+    machine.screenshot("MuseScore1")
+
+    machine.send_key("tab")
+    machine.sleep(1)
+    machine.send_key("tab")
+    machine.sleep(1)
+    machine.send_key("tab")
+    machine.sleep(1)
+    machine.send_key("tab")
+    machine.sleep(1)
+    machine.send_key("right")
+    machine.sleep(1)
+    machine.send_key("right")
+    machine.sleep(1)
+    machine.send_key("ret")
+
+    machine.sleep(1)
 
-    # Start entering notes
-    machine.send_key("n")
     # Type the beginning of https://de.wikipedia.org/wiki/Alle_meine_Entchen
     machine.send_chars("cdef6gg5aaaa7g")
-    # Make sure the VM catches up with all the keys
+
     machine.sleep(1)
 
-    machine.screenshot("MuseScore0")
+    machine.screenshot("MuseScore2")
 
     # Go to the export dialogue and create a PDF
     machine.send_key("alt-f")
     machine.sleep(1)
     machine.send_key("e")
 
     # Wait until the export dialogue appears.
-    machine.wait_for_window("Export")
-    machine.screenshot("MuseScore1")
-    machine.send_key("shift-tab")
-    machine.sleep(1)
+    # machine.wait_for_window("Export")
+    machine.sleep(10)
+
+    machine.screenshot("MuseScore3")
+
     machine.send_key("shift-tab")
     machine.sleep(1)
     machine.send_key("ret")
-    machine.sleep(1)
+
+    machine.sleep(10)
+
     machine.send_key("ret")
 
-    machine.screenshot("MuseScore2")
+    machine.screenshot("MuseScore4")
 
     # Wait until PDF is exported
-    machine.wait_for_file("/root/Documents/MuseScore3/Scores/Untitled.pdf")
+    # machine.wait_for_file("/root/Documents/MuseScore4/Scores/Untitled score.pdf")
```

Comment:
Why is this commented?

Reply:
It doesn’t work, that’s why
I believe the reason is that the space screws up the shell parsing

Reply:
Maybe it can be made to work by adding some escaping, e.g. `machine.wait_for_file("'/root/Documents/MuseScore4/Scores/Untitled score.pdf'")`? Maybe the `wait_for_file` can be fixed for everyone?

Reply:
`…('"…"')` might work. Hold on, I’ll test

Reply:
It works! https://github.com/NixOS/nixpkgs/commit/a67f3fcbc3cfb4640df6feb75f1878ffd5f68a02

Reply:
> It works! [a67f3fc](https://github.com/NixOS/nixpkgs/commit/a67f3fcbc3cfb4640df6feb75f1878ffd5f68a02)

Pushed!